core/src/test/scala/org/broadinstitute/dsde/rawls/monitor/CloneWorkspaceFileTransferMonitorSpec.scala

```diff
@@ -123,7 +123,7 @@ class CloneWorkspaceFileTransferMonitorSpec(_system: ActorSystem)
                             destinationBucketName,
                             objectToCopy.getName,
                             Option(destWorkspace.googleProjectId)
-        )
+        )(system.dispatchers.defaultGlobalDispatcher)
```

Comment:
Just an FYI: You can also declare an implicit val of the dispatcher type at the top of the test class, and it will get picked up automatically.
I don't have any issues with this though.